packages/framework/presence/package.json

```diff
@@ -15,11 +15,21 @@
 	"exports": {
 		".": {
 			"import": {
-				"types": "./lib/index.d.ts",
+				"types": "./lib/public.d.ts",
 				"default": "./lib/index.js"
 			},
 			"require": {
-				"types": "./dist/index.d.ts",
+				"types": "./dist/public.d.ts",
+				"default": "./dist/index.js"
+			}
+		},
```

Comment:
Did you try omitting this? What happened?

Reply:
Omitted and seems to be fine